src/App.js

```diff
@@ -1,30 +1,20 @@
-// src/App.js
-import React from 'react';
-import {
-  BrowserRouter as Router, Routes, Route, Link,
-} from 'react-router-dom';
-import HomePage from './components/HomePage';
-import CategoriesPage from './components/CategoriesPage';
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+/* eslint-disable */
+import Books from './components/homePage';
+import Categories from './components/categoriesPage';
+/* eslint-enable */
```

Comment:
- Kindly remove this linter disabling. you should fix this as it complains about how you've imported those files.